### ISSUE TITLE
[Screenshots] Buildpacks under 'How it Works See It in Action' section

### ISSUE THREAD

**Issue by saurabh100ni** [layer5io/layer5#4859]
### Current Behavior
Currently, [every integration](https://layer5.io/cloud-native-management/meshery/integrations/buildpacks) has only two old screenshots of the designer and visualizer in the 'How it Works See It in Action' section.

### Desired Behavior
- Add new images and videos from the latest version of the software ([MeshMap](https://layer5.io/cloud-native-management/meshmap))
- Add screenshots for [Buildpacks](https://layer5.io/cloud-native-management/meshery/integrations/buildpacks) integration.

### Acceptance Tests
- Ensure your screenshot represents the use of a few specific integrations.
- The screenshot from the Designer mode should contain a valid design, refer to [meshery catalog](https://meshery.io/catalog) to use and view published designs.
- Add both Dark Mode and Light Mode screenshots.
- The screenshot should not be a snippet, take the full screenshot of your browser.
- Use Incognito mode on Chrome browser, to ensure your bookmark bar or any other items are not shown in the screenshot/video.

---

<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="24px" align="left" /><h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
-  🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/buttons/community.webp" height="22px" align="left" />Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

**Comment by csengineer1990** [comment]
Hi, sir, @saurabh100ni  can I work on this issue with your guidance and help.?

**Comment by abhijeetgauravm** [comment]
Hey @csengineer1990 Do you have Meshmap access? Have u set up your Local Development Environment? Here you have to add the new screenshots of the designer and visualizer in the 'How it Works See It in Action' section from the latest version of the software ([MeshMap](https://layer5.io/cloud-native-management/meshmap)) for [Buildpacks](https://layer5.io/cloud-native-management/meshery/integrations/buildpacks) integration. You can go through this [docs](https://docs.meshery.io/extensions/meshmap) for better understanding of Meshmap.

**Comment by csengineer1990** [comment]
I have access to MeshMap, and I have set up my local development environment. I will share the new screenshots of the designer and visualizer shortly. 

Sir, do I have to use an existing designs while taking a screenshot?

**Comment by abhijeetgauravm** [comment]
> Sir, I have access to MeshMap, and I have set up my local development environment. I will share the new screenshots of the designer and visualizer shortly.
> 
> Sir, do I have to use an existing designs while taking a screenshot?

@csengineer1990 yeah regarding design you don't need to create your own, you can use published design and clone them and take screenshot of them (You can do that in meshmap).

**Comment by csengineer1990** [comment]
Hi Sir. Please check whether the following screenshot is valid or not. Please guide me if I am doing anything wrong.

![MeshMap1](https://github.com/layer5io/layer5/assets/68468678/fd41211c-b336-4f7c-b033-6a4a7c7aa7e2)


![MeshMap2](https://github.com/layer5io/layer5/assets/68468678/eeec0a60-22a5-47df-b106-b8f0666c0c72)


![MeshMap3](https://github.com/layer5io/layer5/assets/68468678/96e92534-dcdb-4e73-845f-7540b0291a19)

**Comment by csengineer1990** [comment]
Sir, I have attached the screenshot please check if is it acceptable or not.

![MeshMap1](https://github.com/layer5io/layer5/assets/68468678/cbeac1fb-c1bc-4694-aab6-70ea95ac4b9d)

![MeshMap2](https://github.com/layer5io/layer5/assets/68468678/5549e89a-d0ca-4a6d-840a-667b6501b169)

![MeshMap3](https://github.com/layer5io/layer5/assets/68468678/4bb2298c-5818-4fe2-9bfa-f62675903f08)

**Comment by saurabh100ni** [comment]
@Mohith234 //

**Comment by sandramsc** [comment]
> Hi Sir. Please check whether the following screenshot is valid or not. Please guide me if I am doing anything wrong.
> 
> ![MeshMap1](https://user-images.githubusercontent.com/68468678/276927941-fd41211c-b336-4f7c-b033-6a4a7c7aa7e2.png)
> 
> ![MeshMap2](https://user-images.githubusercontent.com/68468678/276928000-eeec0a60-22a5-47df-b106-b8f0666c0c72.png)
> 
> ![MeshMap3](https://user-images.githubusercontent.com/68468678/276928032-96e92534-dcdb-4e73-845f-7540b0291a19.png)

@csengineer1990 Thank you for your contribution so far...

- The first of the three screenshots would be invalid as the canvas is empty
- The remaining two would be valid, however, it might be that there are icons missing within the shapes rendered on the canvas, although this might be the expected output for the design?
- Reminder to take a screenshot of the **vizualizer** as well.

**Comment by csengineer1990** [comment]
> > Hi Sir. Please check whether the following screenshot is valid or not. Please guide me if I am doing anything wrong.
> > ![MeshMap1](https://user-images.githubusercontent.com/68468678/276927941-fd41211c-b336-4f7c-b033-6a4a7c7aa7e2.png)
> > ![MeshMap2](https://user-images.githubusercontent.com/68468678/276928000-eeec0a60-22a5-47df-b106-b8f0666c0c72.png)
> > ![MeshMap3](https://user-images.githubusercontent.com/68468678/276928032-96e92534-dcdb-4e73-845f-7540b0291a19.png)
> 
> @csengineer1990 Thank you for your contribution so far...
> 
>     * The first of the three screenshots would be invalid as the canvas is empty
> 
>     * The remaining two would be valid, however, it might be that there are icons missing within the shapes rendered on the canvas, although this might be the expected output for the selected design?
> 
>     * Reminder to take a screenshot of the **visualizer** as well.

Ok mam I will check again and I will add a screenshot for the visualizer.

**Comment by csengineer1990** [comment]
ok mam I will check again and I add a screenshot for the visualiser.

**Comment by csengineer1990** [comment]
Hi Sir/Madam, I can't take the screen shoot in visualizer mode because I am facing the issue "Could not load MeshMap GraphQL plugin.". I tried to solve it but couldn't solve it.

I am attaching a few new screenshots which I took in the designed mode with the new design. Please check whether the following screenshot is valid or not. Please guide me if I am doing anything wrong?

![Screenshot 2023-12-17 192130](https://github.com/layer5io/layer5/assets/68468678/3bca9a35-cf80-4d2a-8564-e342777362e3)
![Screenshot 2023-12-17 192116](https://github.com/layer5io/layer5/assets/68468678/9f57e343-b580-4c1a-941a-6fdf187a46e8)
![Screenshot 2023-12-17 191823](https://github.com/layer5io/layer5/assets/68468678/2bf93b2b-4f3f-44ea-8ddd-bb5be7797702)
![Screenshot 2023-12-17 191806](https://github.com/layer5io/layer5/assets/68468678/c360d04a-ddc5-47bb-9499-4bd0a26918f6)
![Screenshot 2023-12-17 191722](https://github.com/layer5io/layer5/assets/68468678/b6b7e8cc-8e9e-4f25-a6b5-884adf7f1bb3)
![Screenshot 2023-12-17 191248](https://github.com/layer5io/layer5/assets/68468678/d7fd6566-2cc3-4455-96cb-dcf595783880)

**Comment by saurabh100ni** [comment]
It should be fixed now, please check again.

**Comment by sandramsc** [comment]
Hi @csengineer1990  update on this issue? Let us know so we know to allow you more time, otherwise, we'll need to free (unassign you) the issue so as to allow other contributors to give it a go.

**Comment by kohantikanath** [comment]
Hi @sandramsc @saurabh100ni,

I would love to work on this issue. Can you please assign it to me?

**Comment by Aarya01Patil** [comment]
Hi @sandramsc @saurabh100ni @hargunkaur286 @narasimha-1511  , I would love to work on this issue. Can you assign it to me?
